src/components/modules/banners/ausMomentThankYouBanner/AusMomentThankYouBanner.tsx

```diff
@@ -735,199 +735,189 @@ const ReadMoreButton: React.FC<ReadMoreButtonProps> = ({
     );
 };
 
-export const AusMomentThankYouBanner: React.FC<BannerProps> = ({
+export const AusMomentThankYouBanner: React.FC<CloseableBannerProps> = ({
     tracking,
     isSupporter,
     tickerSettings,
     countryCode,
-}: BannerProps) => {
+    onClose,
+}: CloseableBannerProps) => {
     if (!(tickerSettings && tickerSettings.tickerData)) {
         return null;
     }
-    const [showBanner, setShowBanner] = useState(true);
     const [isExpanded, setIsExpanded] = useState(false);
 
     const supportersCount = tickerSettings.tickerData.total;
 
     const closeBanner = (): void => {
         setContributionsBannerClosedTimestamp();
```

Comment:
If we moved this into the HOC, it would a) be specific to contributions, which is maybe a bad thing, but b) would remove the need for this function entirely. I imagine all our banners would need to implement something like this.

Reply:
how about:
`const withCloseable = (CloseableBanner: React.FC<CloseableBannerProps>, channel: BannerChannel) ...`
and set the local storage based on the channel

Reply:
🤯 YES! That's such a good idea!

Reply:
I've implemented that now!